lsp/lsp.go

```diff
@@ -245,9 +254,32 @@ func (s *Server) CheckChannels(ctx context.Context, in *breez.CheckChannelsReque
 		return nil, status.Errorf(codes.NotFound, "Not found")
 	}
 	clientCtx := metadata.AppendToOutgoingContext(context.Background(), "authorization", "Bearer "+lsp.Token)
-	reply, err := lspdClient.CheckChannels(clientCtx, &lspdrpc.Encrypted{Data: in.Blob})
+	reply, err := lspdClient.channelOpenerClient.CheckChannels(clientCtx, &lspdrpc.Encrypted{Data: in.Blob})
 	if err != nil {
 		return nil, err
 	}
 	return &breez.CheckChannelsReply{Blob: reply.Data}, nil
 }
+
+func (s *Server) RegisterHtlcNotification(
```

Comment:
NIT: Perhaps a name that exposes higher level of abstraction? `RegisterIncomingPaymentNotification` or just `RegisterPaymentNotification`
Using htlc might make the user think she should get a notification for every htlc.